packages/date-time-picker/README.md

```diff
@@ -0,0 +1,36 @@
+## Description
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/date-time-picker?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/date-time-picker)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/date-time-picker?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/date-time-picker)
+
+```
+yarn add @spectrum-web-components/date-time-picker
+```
+
+Import the side effectful registration of `<sp-date-time-picker>` via:
+
+```
+import '@spectrum-web-components/date-time-picker/sp-date-time-picker.js';
+```
+
+When looking to leverage the `DateTimePicker` base class as a type and/or for extension purposes, do so via:
+
+```
+import { DateTimePicker } from '@spectrum-web-components/date-time-picker';
+```
+
+## Example
+
+```html
+<sp-date-time-picker></sp-date-time-picker>
+```
+
+## To-do list
```

Comment:
Can we remove this?

Reply:
I don't think it's the time to remove them. I will remove them incrementally once they are completed.